src/application/utils/utxos.ts

```diff
@@ -1,22 +1,149 @@
-import { UnblindedOutput, unblindOutput } from 'ldk';
-import { Transaction } from 'liquidjs-lib';
+import {
+  AddressInterface,
+  getNetwork,
+  IdentityInterface,
+  NetworkString,
+  UnblindedOutput,
+  unblindOutput,
+} from 'ldk';
+import { address, networks, Transaction } from 'liquidjs-lib';
+import { RawHex } from 'marina-provider';
+import { RootReducerState } from '../../domain/common';
 import { UnconfirmedOutput } from '../../domain/unconfirmed';
+import { selectNetwork } from '../redux/selectors/app.selector';
+import {
+  selectAllAccounts,
+  selectAllAccountsIDs,
+  selectUtxos,
+} from '../redux/selectors/wallet.selector';
 
```

Comment:
not entirely convinced we should import redux methods in an util method, just for separate concerns and re-utilize the utils in other context? What if passing clousure as parameters so we can pass it from the top

toughts @louisinger?

Reply:
+1, better to keep selectors outside of functions (mainly to be able to launch them in the `container` files for UI)

Reply:
Fixed by c900ca6